### DISCUSSION TITLE
AlienFX Support for AW x17 R2

### DISCUSSION THREAD

**Discussion by XRone99** [T-Troll/alienfx-tools#193] [Q&A]
In the supported device list my laptop is present so I don't understand where I am doing wrong. Does anyone know how to properly set up the tool? It seems to be a very well made tool and I really would like to use it properly.

If someone has had the same problem and maybe figured out how to fix it that would be great

**Comment by T-Troll** [comment]
Yes, x15/17R2 supported.

But no one share light mapping for me, so you need to configure lights manually.
Read [beginners how-to](https://github.com/T-Troll/alienfx-tools/wiki/How-to-start-(Beginner's-guide)-for-release-v6.x.x.x) about how to do it.

This tool is more complex, then AWCC. It's for enthusiasts.

**Reply by XRone99** [reply]
So... in theaory I should be able to configure it manully by myself and in that case I could share the configuration with you
I'll try to do it but it's hard to find the correct configuration

**Reply by T-Troll** [reply]
No so hard. Just take some time. And yes, you can share for other guys like you.
And a bit tricky to configure out what grid dimensions you need for keyboard.

**Reply by XRone99** [reply]
[Current.csv](https://github.com/T-Troll/alienfx-tools/files/9463623/Current.csv)

This is what I managed to get, the only problem is that I still cannot control the Power Botton properly, probably my fault

**Reply by T-Troll** [reply]
Ok, thanks, let me check. 
Power button, logo and tron at other (2nd) device - your system have 2 light control chips (one Darfon for keyboard and one Alienware for tron).

**Reply by XRone99** [reply]
Yes, I noticed that there are two differenti Chips One for the keyboard and the other for PC lights, i manager ti configure all of them but the only One that Is not properly responsive remains the Power button while all other stuff seems to work well

**Reply by T-Troll** [reply]
Well... Hardware power button does not change it color at test, it can only blink.
If you find it's Light ID - just check "Power button" for it - and it will work correctly (but only can have 2 colors - AC/Battery).

PS: I check your file... Seems like you type all names yourself, instead of using "press a key" feature. As a result, key press grid effects will work incorrectly. But, anyway, better than nothing!

**Reply by T-Troll** [reply]
I also see you are use m15R1 profile for tron. Big mistake, you have different light IDs!
Please reset it for all lights (especially power button!) and check manually as well!

**Reply by XRone99** [reply]
Well i didn,t know that the Power Key can only have 2 colours, maybe it's already working how's supposed ti work.
About the "Press the key" Things i used It for most of the keys but for some keys It Just didn't get the input so i had to type It myself
For what concerns the m15R1 profile I didn't notice that It selected by itself on the First openinf of the app
I'll start over later and let you know

**Reply by T-Troll** [reply]
About power key - if you mark power key as power, it will be later configured out at "Colors" tab (for both AC and bat), but this checkbox hint my app how to handle it to work correctly.

Press key... Seems like it's a bit my fault. You have localized version of Windows, so key names is the mess. Thank you for point it.
